tools/osx/crosstool/wrapped_clang.cc

```diff
@@ -118,15 +118,24 @@ void RunSubProcess(const std::vector<std::string> &args) {
   }
 }
 
-// Splits txt using whitespace delimiter, pushing each substring into strs.
+// Splits txt on the first whitespace delimiter, pushing both substring into strs.
+// This is because the wrapped_clang binary is called with arguments like:
+//     wrapped_clang '-isysroot a/path/that might have/spaces' and we need to
```

Comment:
Good point. I'm assuming this is called by Bazel right? I grepped for `isysroot` and it seems Bazel does use a single space:

```
tools/osx/crosstool/CROSSTOOL.tpl:        flag: "-isysroot %{sdk_dir}"
```

for example. Is it the intent to have it be general purpose and support all the different syntax? Note that this might be really hard, for instance in the `-isysroota/path` case, how do you know where the flag ends and the path starts without knowing all of clang's flags?

Also note the previous version would fail in similar or worse ways with either of these other syntaxes.